doc/develop/extending.rst

```diff
@@ -315,9 +315,13 @@ scripts, or via a command-line interface.
    your preference).
 
 #. Create an empty text file `odemis.pth` in the Anaconda Python installation folder:
-   ``C:\Users\YOURUSERNAME\Anaconda2\Lib\site-packages``. Make sure the file does
-   *not* have a `.txt` extension. Edit that file and enter the full path to the
+   ``C:\Users\YOURUSERNAME\AppData\Local\Continuum\anaconda2\Lib\site-packages``
+   (if you cannot find it, check using the console in Spyder with ``import numpy``,
+   then ``numpy.__file__``). Make sure the file does
+   *not* have a `.txt` extension (which the File Explorer automatically hides).
+   Edit that file and enter the full path to the
```

Comment:
Do you mean `Open the file, write the full path ...... and save the file.`

Reply:
Yes, that.
I've just updated the changes :-)